src/index.html

```diff
@@ -31,7 +31,149 @@
     </nav>
   </header>
 
-  <main></main>
+  <main>
+    <div class="p-keyvisual">
+      <div class="p-keyvisual__box">
+        <div class="p-keyvisual__box__text">
+          <img class="p-keyvisual__box__text__img is-noimg" src="./img/icon//keyvisual-no1.png" alt="">
+          <img class="p-keyvisual__box__text__img is-textimg" src="./img/icon//keyvisual-text1.png" alt="">
```

Comment:
[must]
テキストを画像として扱う時は、alt属性に画像に表示されているテキストを入れるようにお願いします！

```suggestion
          <img class="p-keyvisual__box__text__img is-noimg" src="./img/icon//keyvisual-no1.png" alt="2021年使って良かった転職サイト no.1">
          <img class="p-keyvisual__box__text__img is-textimg" src="./img/icon//keyvisual-text1.png" alt="医療事務求人を探すならギズジョブ">
```

代替テキストの設定をすることで、画像が表示されなかった時でもユーザーがコンテンツの内容を理解することができるよになります。